src/app/compliance/overview/compliance-overview-client.tsx

```
@@ -82,7 +82,9 @@ export default function ComplianceOverviewClient({ jurisdiction, organizationId
     return (
       <div className="container mx-auto p-4 sm:p-6 space-y-6">
         <div className="flex items-center justify-center h-64">
-          <div className="text-lg text-red-600">Error loading data</div>
+          <div className="text-lg text-red-600">
+            We couldn't load the compliance overview. Refresh the page to try again.
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&`#39`;`, `&rsquo;`.

(react/no-unescaped-entities)